extensions/gradientbottombar@julienpecqueur.net/schemas/org.gnome.shell.extensions.gradientbottombar.gschema.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I started wondering if we should simply avoid including the `settings.mk` in gradientbottombar Makefile.am instead of adding this file.